requirements/dev.txt

```diff
@@ -33,6 +33,14 @@ transifex-client==0.12.4
 # Dependency of transifex-client
 urllib3==1.20
 
+# Needed for documentation_crawler
+pathlib2==2.2.1
+#Needed for pathlib2==2.2.1
+scandir==1.5
+
```

Comment:
To explain, `pathlib2` is required in dev env, and the only case when it is not installed at all is when building for py2-prod. note: currently expanding the build matrix to cover Pypy, which runs Django 20x faster[1].

[1] http://speed.pypy.org/ (reference snapshot-ed and opentimestamped here https://github.com/rht/opentimestamped/tree/master/speed.pypy.org for scholarly purpose)